playground/pages/[productId]/success.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="flex justify-center items-center flex-col">
+    <h1 class="text-4xl mb-4">
+      Congratulations ! you finalize the subscription
+    </h1>
+    <div v-if="pending">
+      Loading
+    </div>
+    <div v-else-if="error">
+      Error {{ error.message }}
+    </div>
+    <div v-else>
+      <div
+        class="relative"
+        :style="{
+          backgroundColor: 'gray',
+          width: '300px',
+          height: '500px',
+          'background-image': 'url(' + data.images[0] + ')',
```

Comment:
```suggestion
          'background-image': `url('${data?.images[0]')`,
```
data can be undefined, maybe better to return from setup